calico/security/tutorials/calico-policy.md

```diff
@@ -76,7 +76,7 @@ kind: GlobalNetworkPolicy
 metadata:
   name: default-deny
 spec:
-  selector: projectcalico.org/namespace != "kube-system"
+  selector: projectcalico.org/namespace != "kube-system" && projectcalico.org/namespace != "calico-system" && projectcalico.org/namespace != "calico-apiserver" 
```

Comment:
If we need the apiserver included it's likely because we don't have proper egress rules in the policy we already create for it. We should fix that.

Reply:
given you're doing multiple values for the same key, we should use a simplified selector format:


```
selector: projectcalico.org/namespace not in ["kube-system", "calico-system", "calico-apiserver"]
```

Also, we need to explain why we're excluding those namespaces.

e.g., "Exclude some well-known namespaces, since we don't want to block communication that is essential for a functioning cluster"